### DISCUSSION TITLE
How do I change the font size for sub level bullets in a list

### DISCUSSION THREAD

**Discussion by CMFell** [quarto-dev/quarto-cli#7626] [Q&A] [answered]
### Description

How do I change the font size for some bullet points within a list
In particular, I want second level bullets to be smaller.

````{qmd}
---
title: "example"
format: revealjs
---

## Slide with bullets

::: {.incremental}

* Bullet 1
* Bullet 2
    + This text should have a smaller font size
* Another bullet

:::

````

**Comment by mcanouil** [comment, marked as answer]
Use CSS and/or span/divs, see <https://quarto.org/docs/authoring/markdown-basics.html#divs-and-spans>.

````qmd
---
title: "example"
format: revealjs
---

## Slide with bullets

::: {.incremental}

* Bullet 1
* Bullet 2
    + [This text should have a smaller font size]{style="font-size:0.5em"}
* Another bullet;

:::
```` 

or 

````qmd
---
title: "example"
format: revealjs
include-in-header:
  - text: |
      <style type="text/css">
      ul li ul li {
        font-size: 0.5em;
      }
      </style>
---

## Slide with bullets

::: {.incremental}

* Bullet 1
* Bullet 2
    + This text should have a smaller font size
* Another bullet;

:::
````

Or plenty other CSS ways.

**Reply by CMFell** [reply]
Brilliant, the second method is exactly what I was looking for.

I had been trying ::: {.smaller} ::: around the bullet points and getting odd results.

**Reply by mcanouil** [reply]
Note that the second method applies to absolutely everything.

**Reply by JosiahParry** [reply]
@mcanouil does the syntax `[This text should have a smaller font size]{style="font-size:0.5em"}` create a span? Is there a supported way of writing inline spans? For example I might want to add a style to R package names that i've written in line

**Reply by mcanouil** [reply]
Yes, this is described in the markdown basics guide, see <https://quarto.org/docs/authoring/markdown-basics.html#divs-and-spans>.

PS: I do not understand your "inline span" comment. Spans are by definition inline as opposed to divs.

**Reply by cscheid** [reply]
@JosiahParry There's a trick you can use to check what Pandoc does:

```
$ cat file.md
[This text should have a smaller font size]{style="font-size:0.5em"}
$ quarto pandoc -f markdown -t html file.md
<p><span style="font-size:0.5em">This text should have a smaller font
size</span></p>
```

The paragraph tag is there simply because Pandoc doesn't like putting a `<span>` in the main document by itself:

```
$ cat file2.md
Some more text, and then [This text should have a smaller font size]{style="font-size:0.5em"}
$ quarto pandoc -f markdown -t html file2.md
<p>Some more text, and then <span style="font-size:0.5em">This text
should have a smaller font size</span></p>
```

You can also opt to inspect the "native" Pandoc AST, which is not rendered to any particular format:

```
$ quarto pandoc -f markdown -t native file2.md
[ Para
    [ Str "Some"
    , Space
    , Str "more"
    , Space
    , Str "text,"
    , Space
    , Str "and"
    , Space
    , Str "then"
    , Space
    , Span
        ( "" , [] , [ ( "style" , "font-size:0.5em" ) ] )
        [ Str "This"
        , Space
        , Str "text"
        , Space
        , Str "should"
        , Space
        , Str "have"
        , Space
        , Str "a"
        , Space
        , Str "smaller"
        , Space
        , Str "font"
        , Space
        , Str "size"
        ]
    ]
]
```
